src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -1763,7 +1760,13 @@ internal byte[] SerializeFloat(float v)
                 throw ADP.ParameterValueOutOfRange(v.ToString());
             }
 
+#if NETCOREAPP
+            var bytes = new byte[4];
+            BinaryPrimitives.WriteInt32LittleEndian(bytes, BitConverter.SingleToInt32Bits(v));
+            return bytes;
```

Comment:
Is there a way to do this using the span we already have instead of allocating a new byte[]? If an array is needed it could be stackalloc-ed since we know it is fixed size.